content/id/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -0,0 +1,109 @@
+---
+title: Menggunakan Service untuk Ekpos Aplikasimu
```

Comment:
Typo? and please use consistently between "ekspos" and "mengekspos"
```suggestion
title: Menggunakan Service untuk Mengekspos Aplikasimu
```